### ISSUE TITLE
DashBoard Showing wrong count

### ISSUE THREAD

**Issue by gvreddyreddy** [adiralashiva8/testng-metrics#18]
I have executed only 1 test case but Dashboard showing count as 4
```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite name="Forgot Password">
	<test name="Forgot Password" verbose="2">
		<parameter name="browserName" value="chrome" />
		<classes>
			<class name="com.fs.goeasy.tests.ForgotPwdTest">
			<methods>
			 <include name="tc05_validateErrorMsgForPasswordsMismatch"></include>
			</methods>
			</class>
		</classes>
	</test>
</suite>
```
[Metrics-2019Feb28-1310.zip](https://github.com/adiralashiva8/testng-metrics/files/2913914/Metrics-2019Feb28-1310.zip)

**Comment by gvreddyreddy** [comment]
More info :  In my Test Case i have 5 @Test method but i am executing only one like following .

<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite name="Forgot Password">
	<test name="Forgot Password" verbose="2">
		<parameter name="browserName" value="chrome" />
		<classes>
			<class name="com.fs.goeasy.tests.ForgotPwdTest">
			<methods>
			 <include name="tc05_validateErrorMsgForPasswordsMismatch"></include>
			</methods>
			</class>
		</classes>
	</test>
</suite>

**Comment by gvreddyreddy** [comment]
Sample TestNG xml file.
[sample.zip](https://github.com/adiralashiva8/testng-metrics/files/2914186/sample.zip)

**Comment by gvreddyreddy** [comment]
**One more issue:** 
To show total count in code observed that you have written following code
int getTotal() {
      return getPassed() + getFailed() + getSkipped() + getIgnored() + getRetried();
    }

Here ignored and retired count also but not showing in the report so we are unable to understand report .

If we show ignored and retired count in report then count will match.

**Comment by adiralashiva8** [comment]
Thank you @gvreddyreddy for pointing the issue and also provding solution for it. will fix this as part of v1.4 release

**Comment by adiralashiva8** [comment]
Fixed as part of https://github.com/adiralashiva8/testng-metrics/commit/807137d18b040c3a35cea227976fddd398749124 commit
